Doc/library/plistlib.rst

```diff
@@ -44,10 +44,16 @@ or :class:`datetime.datetime` objects.
 .. versionchanged:: 3.9
    Old API removed.
 
+.. versionchanged:: 3.10
```

Comment:
This should be bumped to 3.11 (or based on how slowly this is getting reviewed, maybe 3.12 😢 )

Reply:
```suggestion
.. versionchanged:: 3.13
```